src/Avalonia.Controls/Primitives/SelectingItemsControl.cs

```diff
@@ -209,6 +225,27 @@ public int SelectedIndex
             }
         }
 
+        /// <summary>
+        /// Gets the <see cref="IBinding"/> instance used to obtain the 
+        /// <see cref="SelectedValue"/> property
+        /// </summary>
+        [AssignBinding]
+        public IBinding? SelectedValueBinding
```

Comment:
It would be great if this property supported compiled bindings without need to specify x:DataType.

I hope [InheritDataTypeFromItems](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/ItemsControl.cs#L171) will just work here. And double checked if it works.

Reply:
Oh, I completely missed that being added. Will add that attribute and test it works